### DISCUSSION TITLE
[Support]  midway/koa  serverTimeout超时时间设置无效，总是5分钟超时

### DISCUSSION THREAD

**Discussion by shuaishenk** [midwayjs/midway#3794] [Q&A] [answered]
### Describe the problem（描述问题）

![image](https://github.com/midwayjs/midway/assets/3381855/dc3edc8e-f66b-407f-9f53-5ff37f7fb183)
![image](https://github.com/midwayjs/midway/assets/3381855/bbb200ae-5563-407d-959c-2860f65ef95c)

### Midway Versions（Midway 版本）

3.11.1

**Comment by czy88840616** [comment]
这个 5 分钟是 requestTimeout 的报错。看起来还是得设置 server.requestTimeout

**Comment by czy88840616** [comment, marked as answer]
已经发新版并更新文档章节 https://midwayjs.org/docs/extensions/koa#%E8%B6%85%E6%97%B6%E9%85%8D%E7%BD%AE

**Reply by shuaishenk** [reply]
这效率，给大佬们点赞
